frappe/public/js/frappe/form/controls/base_input.js

```diff
@@ -101,6 +103,16 @@ frappe.ui.form.ControlInput = class ControlInput extends frappe.ui.form.Control
 					}
 				}
 				me.$input && me.$input.prop("disabled", true);
+
+				if (is_fetch_from_read_only) {
+					$(me.disp_area).attr(
+						"title",
+						__(
+							"Its value is getting fetched from {0}'s {1} field",
```

Comment:
```suggestion
							"This value is fetched from {0}'s {1} field",
```

If we could do labels it would be great but that'd likely require fetching linked doctype's meta? :thinking: